src/Components/Book.js

```diff
@@ -1,15 +1,24 @@
-import React from 'react';
+// import React from 'react';
+// import { useDispatch } from 'react-redux';
+// import { RemoveBook } from '../redux/books/bookSlice';
 
-const Book = (props) => {
-  return (
-    <React.Fragment>
-      <h1>{props.book.title}</h1> 
-      <h3>{props.book.author}</h3>
-      <button type='button'>Comments</button>
-      <button type='button'>Remove</button>
-      <button type='button'>Edit</button>
-    </React.Fragment>
-  );
-};
+// const Book = (props) => {
+//   const dispatch = useDispatch;
+//   const onSaveDelete = () => {
+//     dispatch(
+//       RemoveBook()
+//     )
+//   }
 
-export default Book;
+//   return (
+//     <React.Fragment>
+//       <h1>{props.renderedBook.title}</h1> 
+//       <h3>{props.renderedBook.author}</h3>
+//       <button type='button'>Comments</button>
+//       <button type='button' onClick={onSaveDelete}>Remove</button>
+//       <button type='button'>Edit</button>
+//     </React.Fragment>
+//   );
+// };
+
+// export default Book;
```

Comment:
- [ ] You are almost there! It would be great to remove any unused code to keep your code clean.

Reply:
Thank you
I have noticed the issue. I am fixing it now

Reply:
This issue is well resolved! ✔️

Reply:
Thank you,
I would be waiting for the approved message 😉

Reply:
Nice job! 😎